rules/sample_cpg_pileup.smk

```diff
@@ -0,0 +1,26 @@
+rule cpg_pileup:
+    input:
+        bam = f"samples/{sample}/whatshap/{sample}.{ref}.deepvariant.haplotagged.bam",
+        bai = f"samples/{sample}/whatshap/{sample}.{ref}.deepvariant.haplotagged.bam.bai"
+        reference = config['ref']['fasta'],
+    output:
+        [f"samples/{sample}/cpg_pileup/{sample}.{ref}.{infix}.denovo.{suffix}"
+         for infix in ['combined', 'hap1', 'hap2']
+         for postfix in ['bed', 'bw', 'mincov4.bed', 'mincov4.bw']]
+    log: f"samples/{sample}/logs/cpg_pileup/{proband}.{ref}.log"
```

Comment:
Shouldn't it be {sample}.{ref}.log, proband isn't defined?

Reply:
Nice catch, thanks.